main.go

```diff
@@ -246,3 +256,19 @@ func (rs RepoSync) Sync() error {
 	cloners.Wait()
 	return nil
 }
+
+func monthsCountSince(createdAtTime time.Time) int {
```

Comment:
nit: `t` instead of `createdAtTime` since this function is not specific to created time